.build/Dockerfile

```diff
@@ -6,4 +6,8 @@ RUN apt-get -y update \
     && apt-get -y clean \
     && rm -rf /var/lib/apt/lists/*
 
-RUN go get github.com/google/googet/goopack
+RUN GO111MODULE=on go get github.com/client9/misspell/cmd/misspell \
```

Comment:
You could probably define the environment variable separately to avoid repeating it:
```
ENV GO111MODULE=on
RUN go get github.com/client9/misspell/cmd/misspell \
  && go get github.com/golangci/golangci-lint/cmd/golangci-lint \
  ...
```